src/Misc/layoutbin/en-US/strings.json

```diff
@@ -340,7 +342,7 @@
   "SelfManageGitCreds": "You are in self manage git creds mode. Make sure your agent host machine can bypass any git authentication challenge.",
   "ServerTarpit": "The job is currently being throttled by the server. You may experience delays in console line output, job status reporting, and task log uploads.",
   "ServerUrl": "server URL",
-  "ServerUrlForMachineGroupAgent": "server URL ( Provide project level server url e.g https://myaccount.visualstudio.com/myproject)",
+  "ServerUrlForMachineGroupAgent": "Enter server URL (e.g for VSTS: https://myaccount.visualstudio.com/myproject, for TFS: http://mylocaltfsserver:8080/tfs/mycollection/myproject)",
```

Comment:
is this required? IMO we should not do this - what if VSTS suddently decides to include Collection in the URL?